pkg/rc/replication_controller.go

```diff
@@ -902,17 +910,60 @@ func (rc *replicationController) doBackgroundNodeTransfer() {
 
 	rc.logger.Infof("Node transfer from %s to %s complete.", rc.nodeTransfer.oldNode, rc.nodeTransfer.newNode)
```

Comment:
This log I wrote is actually misleading since the node transfer may not have been successful. I didn't catch that until your changes